en/docs/design/rate-limiting/resetting-application-throttling-policies.md

```diff
@@ -0,0 +1,30 @@
+# Reset Application Throttling Policies
+
+WSO2 API Manager allows you to reset the Application-level rate limiting tier for a specified end-user. This feature can be used by the application owner or shared owner to reset throttle policies by providing the username of the application user on their request. 
+
+[![Application Overview]({{base_path}}/assets/img/learn/application-overview-reset.png)]({{base_path}}/assets/img/learn/application-overview-reset.png)
+
+For each application user, a counter is maintained at the traffic manager of API Manager. When the counter reaches the limit defined by the application throttle policy, the relevant application user will be throttled out and further API invocation will be restricted. If the user wants to continue invoking APIs using the application with same application throttle policy, the application owner can accommodate that by resetting the quota used by that specific user only.
+
+1. Fill in the **username** of the user whose quota needs to be reset and click **Next**.
+
```

Comment:
This should be "username or uuid" right?

Reply:
Check at all the places you had mentioned "username". I have pointed only few places

Reply:
Fixed with https://github.com/wso2/docs-apim/pull/7990/commits/880dbfccec85687455ced5e2fa4c751e1a5333d7